### ISSUE TITLE
Clean-up warning due to unused generic type parameter

### ISSUE THREAD

**Issue by rbeeli** [JuliaDatabases/ClickHouse.jl#34]
With the latest version of **ClickHouse.jl**, we get some ugly warnings due to unused type variables. Could those be removed? Should be a rather easy fix.

```
 ClickHouse [82f2e89e-b495-11e9-1d9d-fb40d7cf2130]
│  WARNING: method definition for write_col_data at /home/rino/.julia/packages/ClickHouse/j7XeK/src/columns/Nullable.jl:92 declares type variable T but does not use it.
│  WARNING: method definition for insert at /home/rino/.julia/packages/ClickHouse/j7XeK/src/Query.jl:80 declares type variable T but does not use it.
└  
```

Thanks.

**Comment by athre0z** [comment]
I don't really use Julia anymore and the other maintainer, @waralex, hasn't had any activity on GitHub for about a year. I'm still happy to review PRs though.

**Comment by quinnj** [comment]
Fixed in #35
